### PR TITLE
Unified recipe WIP

### PR DESCRIPTION
**Pull request by hobu**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by hobu** [comment]
This doesn't work at this time.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -50,13 +51,112 @@ requirements:
...
+        dev_url: https://github.com/pdal/pdal
+
+    - name: pdal
+      version: {{ python_version }}
+      source:
+        fn: "python-pdal-{{ python_version }}.tar.gz"
+        url: https://github.com/PDAL/python/archive/{{ python_version }}.tar.gz
+        sha256: {{ python_sha256 }}
+
+        skip: true  # [(win and vc<14) or (py<36) or (py<38 and linux)]
```
Comment (weiji14) [inline recipe/meta.yaml:125]: You can't add an extra `source` here. Also, why are you trying to build a python-pdal here when https://github.com/conda-forge/python-pdal-feedstock already exists?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -50,13 +51,112 @@ requirements:
     - libxml2
     - draco
     - openssl
-  run_constrained:   # [osx and x86_64]
-    - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.9") }}  # [osx and x86_64]
+    - pdal
+
+  outputs:
```
Comment (weiji14) [inline recipe/meta.yaml:56]: This should be at the first indent.

```suggestion
outputs:
```

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,17 +1,18 @@
...
 
 build:
-  number: 2
-  run_exports:
-    - {{ pin_subpackage('pdal', max_pin='x.x') }}
+  number: 0
+  skip: true  # [(win and vc<14) or (py<36) or (py<38 and linux)]
 
 requirements:
   build:
```
Comment (weiji14) [inline recipe/meta.yaml:18]: If using `outputs:`, then all the lines for requirements (L17 to L54) need to be removed.

**Review by weiji14 — COMMENTED** [review]
Trying to understand the intent of this PR. Are you trying to rename the PDAL C libary from `pdal` -> `libpdal`, and the Python library from `python-pdal` -> `pdal`? If so, you will need to communicate this to your users, and follow the renaming guidelines at https://conda-forge.org/docs/orga/guidelines.html#renaming-packages. Downstream packages that have listed `pdal` as their dependency will need to be informed about the change to `libpdal`, otherwise they will be surprised as to why Python was suddenly pulled in. Oh, and the feedstock at https://github.com/conda-forge/python-pdal-feedstock will also need to be archived.

Alternatively, you can keep the current `pdal` (C library) and `python-pdal` (Python library) distinction.

File: recipe/meta.yaml (hunk 1 above)
Comment (hobu) [reply recipe/meta.yaml:125]: I was trying to see if we could consolidate the package into a libpdal/pdal package like GDAL's. This PR is mostly experimentation, and I appreciate the feedback.

One of the issues as you point out is the PDAL python bindings are a separate release from PDAL, so for now I have given up on this effort.

**Pull request closed without merging**